sw/device/lib/testing/otbn_testutils.c

```diff
@@ -0,0 +1,99 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/otbn_testutils.h"
+
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_base.h"
+#include "sw/device/lib/dif/dif_otbn.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+
+/**
+ * Data width of big number subset, in bytes.
+ */
+const int kOtbnWlenBytes = 256 / 8;
```

Comment:
nit: if you don't need the address of this consider an anonymous enum (and `CHAR_BIT` instead of `8`):
```
enum {
    kOtbnWlenBytes = 256 / CHAR_BIT;
};

Reply:
I put it in an enum, but am still using 8 because `CHAR_BIT` is undefined unless I import `freestanding/limits.h`, and I'm not sure I should do that here.